inception-kb/pom.xml

```diff
@@ -128,6 +128,10 @@
       <artifactId>rdf4j-sparqlbuilder</artifactId>
       <version>${rdf4j.version}</version>
     </dependency>
+    <dependency>
+      <groupId>com.github.jsonld-java</groupId>
+      <artifactId>jsonld-java</artifactId>
```

Comment:
Why do we need jsonld?

Reply:
The RDF4J query builder somehow depends on it. If I exclude the dependency, the compile fails. If I do not include it explicitly, then the dependency plugin complains. It's somehow used by some anonymous lambda. I didn't bother to investigate in detail. We have it as a transitive anyway.